vmproxy/proxy/proxy.go

```diff
@@ -88,12 +88,9 @@ func director(target *url.URL, headerName string) func(*http.Request) {
 			q := req.URL.Query()
 			q.Del("extra_filters[]")
 
-			parsed, err := parseFilters(filters)
-			if err != nil {
```

Comment:
All errors are logged in `parseFilters`, this was redundant

Reply:
It seems like removing logging from there would make more sense, otherwise there is no reason to return an error at all.

Reply:
also we could wrap the errors we got in `parseFilters`

Reply:
> also we could wrap the errors we got in `parseFilters`

Great suggestion, it's done.